lua/starfall/libs_sh/string.lua

```diff
@@ -14,6 +14,19 @@ local col_meta, cwrap, cunwrap = instance.Types.Color, instance.Types.Color.Wrap
 local string_library = instance.Libraries.string
 local sfstring = SF.SafeStringLib
 
+if CLIENT then
```

Comment:
This can be shared

Reply:
GLua's [`markup.Escape`](https://wiki.facepunch.com/gmod/markup.Escape) is clientside only. I can just copy it's source and sanitize the string myself, but then it would have to be updated manually.
Do you still want me to make it shared?

Reply:
Sure, that's fine